deep/settings.py

```diff
@@ -81,8 +81,10 @@
     ELASTI_CACHE_PORT=str,
     # UNHCR Token
     UNHCR_PORTAL_API_KEY=(str, None),
+    MAX_PAGE_NUMBER=(int, 100)
 )
 
+MAX_PAGE_NUMBER = env('MAX_PAGE_NUMBER')
```

Comment:
This is not required, let's just define this directly in the base.

NOTE: Also, make sure to use a prefix when defining global variables like this. MAX_PAGE_NUMBER is a generic name. eg: UNIFIED_CONNECTOR_SOURCE_MAX_PAGE_NUMBER